static/modes/v-mode.ts

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) 2012, Compiler Explorer Authors
```

Comment:
wrong date

Reply:
fixed in [49c6438](https://github.com/compiler-explorer/compiler-explorer/pull/5297/commits/49c6438c7ca7932b4acc7a0ab5b35bec117d78e0)